src/BackendStatusBanner/index.jsx

```diff
@@ -17,12 +20,40 @@ class BackendStatusBanner extends React.Component {
     this.props.pingStudio();
   }
 
+  renderStatusMessage() {
+    const status = statusMap[this.props.connectionStatus];
+    if (status.link) {
+      return (
+        <span>
+          {status.message}
+          {' '}
+          <a href={status.link.url} className={styles['alert-link']}>{status.link.text}</a>
+        </span>
+      );
+    }
+    return status.message;
+  }
+
   render() {
-    return (this.props.connectionStatus === 200) ?
+    const status = statusMap[this.props.connectionStatus];
+    return (!status || this.props.connectionStatus === 200) ?
       null :
       (
-        <div className="api-error">
-          {statusMap[this.props.connectionStatus]}
+        <div
+          className={classNames(
+            styles['api-error'],
+            styles.alert,
+            styles[`alert-${status.alertLevel}`],
+          )}
+        >
+          <Button
+            display="↻"
```

Comment:
Is this a unicode character that we can use for a button logo? Cool.

Reply:
❤️ how easy this was to declare

Reply:
Yup! I figured that for our dev environments a simple unicode character would do.